lib/Watching.js

```diff
@@ -266,7 +266,11 @@ class Watching {
 					this.watcher.pause();
```

Comment:
I guess we should not pause the watcher when suspended, so it keeps aggregating changes. Otherwise `modifiedFiles` etc. do not contain the changes from the first watch event.

Reply:
I am maybe missing something but it seems to be a dead code anyways, because `this.watcher` is always `null` at this point so this `if` branch with `this.watcher.pause()` is actually never executed (see line 252)?

https://github.com/webpack/webpack/blob/f4ae05ecd62b30e8e7b44cd3e6dd5916bcdb9bc0/lib/Watching.js#L252-L266